IDE/WICED-STUDIO/README

```diff
@@ -0,0 +1,44 @@
+The wolfSSL examples in the WICED SDK require a server or client application. wolfSSL
+provides an easy solution in the <wolfSSL root>/examples/ directory. Run the autogen
+script and configure the library with necessary functionality, see ./configure -h for a
+list of configure options. Finally, run make and make check to ensure the
+build was successfull. You can install wolfSSL on your system with make
+install if you wish. See www.wolfssl.com/docs/ for more information.
```

Comment:
Let's point people specifically to Chapter 2 of the wolfSSL Manual (Building wolfSSL) here.

Reply:
Thanks